Sandbox/Sandbox/Base.lproj/Main.storyboard

```diff
@@ -154,7 +154,7 @@
                                 <textInputTraits key="textInputTraits"/>
                             </textField>
                             <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Redirect URI" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="yXF-2L-fNb">
-                                <rect key="frame" x="20" y="102" width="95.5" height="21"/>
+                                <rect key="frame" x="20" y="102" width="95.666666666666686" height="21"/>
```

Comment:
Why don't you round the number?

Reply:
I don't know what this was changed. I don't control the storyboard directly. Maybe I should just revert this change...

Reply:
It's ok, if XCode did it, then it must have a good reason.

Reply:
I reverted and saw no difference. I pushed the revert.